dsmr_parser/objects.py

```diff
@@ -155,6 +155,16 @@ def __init__(self, values):
         super().__init__(values)
         self._buffer_list = None
 
+    @property
+    def value(self):
+        # value is added to make sure the telegram iterator does not break
+        return self.__str__()
```

Comment:
Are we sure we wanna mix human readable __str__ representations into 'value's of the other objects?

Reply:
I actually don't mind what it returns, as long as it returns something.
What do you think is better?

- None
- the contents of values, for example:
'telegram.POWER_EVENT_FAILURE_LOG.values
[{'value': 3, 'unit': None}, {'value': '0-0:96.7.19', 'unit': None}, {'value': datetime.datetime(2000, 1, 4, 17, 3, 20, tzinfo=<UTC>), 'unit': None}, {'value': 237126, 'unit': 's'}, {'value': datetime.datetime(1999, 12, 31, 23, 0, 1, tzinfo=<UTC>), 'unit': None}, {'value': 2147583646, 'unit': 's'}, {'value': datetime.datetime(2000, 1, 1, 23, 0, 3, tzinfo=<UTC>), 'unit': None}, {'value': 2317482647, 'unit': 's'}]'

The latter might be more clean.
Any other suggestion?

Reply:
The contents of the values seem more suited?